SalesforceCommerceCloud/pwa-kit
# [SDK] Fix and rename this build & watch script

# Description

This npm script `dev:watch` is not working because it's pointing to `dev:build` that no longer exists. So this PR does the following changes:
- Now pointing to `prod:build`
- Rename the script to `prod:build-watch` ← open to better suggestion too

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# How to Test-Drive This PR

1. `cd` to the sdk package directory
2. Run `npm run prod:build-watch`
3. Verify that it's able to build and then watch successfully
4. Make some code change in the SDK
5. Verify that the change automatically triggers a rebuild

# Checklists

## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
